tests/phpunit/api/v3/LocBlockTest.php

```diff
@@ -60,16 +60,28 @@ public function testCreateLocBlock() {
       'street_address' => '1234567',
     ));
     $params = array(
-      'address_id' => $address['id'],
-      'phone_id' => $phone['id'],
-      'email_id' => $email['id'],
+      'address' => array(
```

Comment:
Eileen, I think the issue is that this test was improperly formatted.

Looking at https://github.com/civicrm/civicrm-core/pull/8773/files#diff-d944fc840439db6b8c1d9f5ea26d626cR91 i think you have got it correct IMO